cli.js

```diff
@@ -41,14 +41,14 @@ async function main () {
         if (args.length < 3 || args.length > 4) {
             console.error(
                 'Usage: npm run tds',
-                './tds-input.json ./supported-surrogates-input.json ',
+                './tds-input.json analytics.js,ga.js,gpt.js ',
```

Comment:
I'm not sure that this is an improvement because:
 1. Now we cannot provide 0 surrogates, so to use the CLI we'll have to make up a fake surrogate name to put here.
 2. If the number of surrogates increases, this gets unwieldy to use.

Why not just make the supported surrogates file optional?

Reply:
Fair enough, I don't have time to work on this at the moment but I would accept a PR that added CLI handling of an optional `--surrogates` (or similar) argument.